src/plugins/multimedia/multimedia.js

```diff
@@ -628,11 +638,20 @@ $document.on( youtubeEvent, selector, function( event, data ) {
 			$this = $( event.currentTarget ),
 			$media, ytPlayer;
 
+		// if YouTube on iOS, load YouTube's controls.  Due to restrictions by Apple
+		// and Youtube, videos cannot be played with external controls ( i.e. playVideo() )
+		var $iOS = ( /iPad|iPhone|iPod/.test( navigator.userAgent ) && !window.MSStream );
+		$iOS = true;
```

Comment:
There's an error in the code that needs to be fixed.   The $ios=True line should be removed, it was for testing purposes only.   Will fix.